includes/class-amp-theme-support.php

```diff
@@ -1016,4 +1012,20 @@ public static function prepare_response( $response, $args = array() ) {
 
 		return $response;
 	}
+
+	/**
+	 * Enqueue AMP assets if this is an AMP endpoint.
+	 *
+	 * @since 0.7
```

Comment:
Maybe this should be `1.0` instead. I'm happy to change it.

Reply:
0.7 is correct. I changed the branch to merge this PR.